core/src/storage/storage_db/snapshot_mpt.rs

```diff
@@ -10,6 +10,11 @@ pub trait SnapshotMptTraitReadOnly {
     fn iterate_subtree_trie_nodes_without_root(
         &self, path: &dyn CompressedPathTrait,
     ) -> Box<dyn SnapshotMptIteraterTrait>;
+
+    fn get_manifest(
+        &self, start_chunk: &ChunkKey,
+    ) -> Result<Option<RangedManifest>>;
+    fn get_chunk(&self, key: &ChunkKey) -> Result<Option<Chunk>>;
```

Comment:
Without size?

Reply:
I think the ChunkKey (e.g. include the start and end key prefix) is enough to get the chunk data. I think the chunk size is determined when generating the manifest.